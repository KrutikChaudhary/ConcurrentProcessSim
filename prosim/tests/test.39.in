8 3
Proc1 6 -1
DOOP 5
LOOP 5
  BLOCK -1
  DOOP 4
END
HALT

Proc2 6 -1
DOOP 5
LOOP 4
  BLOCK 2
  DOOP 5
END
HALT

Proc3 6 -1
DOOP 5
LOOP 5
  BLOCK 1
  DOOP 4
END
HALT

Proc4 6 -1
DOOP 5
LOOP 4
  BLOCK 2
  DOOP 5
END
HALT

Proc5 6 -1
DOOP 2
LOOP 5
  BLOCK 5
  DOOP 1
END
HALT

Proc6 6 -1
DOOP 2
LOOP 4
  BLOCK 6
  DOOP 2
END
HALT

Proc7 6 -1
DOOP 2
LOOP 5
  BLOCK 5
  DOOP 1
END
HALT

Proc8 6 -1
DOOP 2
LOOP 4
  BLOCK 6
  DOOP 2
END
HALT

