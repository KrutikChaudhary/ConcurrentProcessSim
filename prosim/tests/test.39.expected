00000: process 1 new
00000: process 1 ready
00000: process 2 new
00000: process 2 ready
00000: process 3 new
00000: process 3 ready
00000: process 4 new
00000: process 4 ready
00000: process 5 new
00000: process 5 ready
00000: process 6 new
00000: process 6 ready
00000: process 7 new
00000: process 7 ready
00000: process 8 new
00000: process 8 ready
00000: process 5 running
00002: process 5 blocked
00002: process 6 running
00004: process 6 blocked
00004: process 7 running
00006: process 7 blocked
00006: process 8 running
00007: process 5 ready
00007: process 8 ready
00007: process 5 running
00008: process 5 blocked
00008: process 8 running
00009: process 8 blocked
00009: process 1 running
00010: process 6 ready
00010: process 1 ready
00010: process 6 running
00011: process 7 ready
00011: process 6 ready
00011: process 7 running
00012: process 7 blocked
00012: process 6 running
00013: process 5 ready
00013: process 6 blocked
00013: process 5 running
00014: process 5 blocked
00014: process 1 running
00015: process 8 ready
00015: process 1 ready
00015: process 8 running
00017: process 7 ready
00017: process 8 blocked
00017: process 7 running
00018: process 7 blocked
00018: process 1 running
00019: process 6 ready
00019: process 5 ready
00019: process 1 ready
00019: process 5 running
00020: process 5 blocked
00020: process 6 running
00022: process 6 blocked
00022: process 1 running
00023: process 8 ready
00023: process 7 ready
00023: process 1 ready
00023: process 7 running
00024: process 7 blocked
00024: process 1 running
00025: process 5 ready
00025: process 1 blocked
00025: process 5 running
00026: process 1 ready
00026: process 5 blocked
00026: process 8 running
00028: process 6 ready
00028: process 8 blocked
00028: process 6 running
00029: process 7 ready
00029: process 6 ready
00029: process 7 running
00030: process 7 blocked
00030: process 6 running
00031: process 5 ready
00031: process 6 blocked
00031: process 5 running
00032: process 5 finished
00032: process 1 running
00034: process 8 ready
00034: process 1 ready
00034: process 8 running
00035: process 7 ready
00035: process 8 ready
00035: process 7 running
00036: process 7 finished
00036: process 8 running
00037: process 6 ready
00037: process 8 blocked
00037: process 1 running
00039: process 1 blocked
00039: process 6 running
00040: process 1 ready
00041: process 6 finished
00041: process 1 running
00043: process 8 ready
00043: process 1 ready
00043: process 8 running
00045: process 8 finished
00045: process 1 running
00047: process 1 blocked
00047: process 2 running
00048: process 1 ready
00048: process 2 ready
00048: process 1 running
00051: process 1 ready
00051: process 1 running
00052: process 1 blocked
00052: process 2 running
00053: process 1 ready
00055: process 2 ready
00055: process 2 running
00056: process 2 blocked
00056: process 1 running
00058: process 2 ready
00059: process 1 ready
00059: process 1 running
00060: process 1 blocked
00060: process 3 running
00061: process 1 ready
00061: process 3 ready
00061: process 1 running
00064: process 1 ready
00064: process 1 running
00065: process 1 finished
00065: process 3 running
00068: process 3 ready
00068: process 3 running
00069: process 3 blocked
00069: process 4 running
00070: process 3 ready
00070: process 4 ready
00070: process 3 running
00073: process 3 ready
00073: process 3 running
00074: process 3 blocked
00074: process 4 running
00075: process 3 ready
00077: process 4 ready
00077: process 4 running
00078: process 4 blocked
00078: process 3 running
00080: process 4 ready
00081: process 3 ready
00081: process 3 running
00082: process 3 blocked
00082: process 2 running
00083: process 3 ready
00083: process 2 ready
00083: process 3 running
00086: process 3 ready
00086: process 3 running
00087: process 3 blocked
00087: process 2 running
00088: process 3 ready
00090: process 2 ready
00090: process 2 running
00091: process 2 blocked
00091: process 3 running
00093: process 2 ready
00094: process 3 ready
00094: process 3 running
00095: process 3 blocked
00095: process 4 running
00096: process 3 ready
00096: process 4 ready
00096: process 3 running
00099: process 3 ready
00099: process 3 running
00100: process 3 finished
00100: process 4 running
00103: process 4 ready
00103: process 4 running
00104: process 4 blocked
00104: process 2 running
00106: process 4 ready
00107: process 2 ready
00107: process 2 running
00109: process 2 blocked
00109: process 4 running
00111: process 2 ready
00112: process 4 ready
00112: process 4 running
00114: process 4 blocked
00114: process 2 running
00116: process 4 ready
00117: process 2 ready
00117: process 2 running
00119: process 2 blocked
00119: process 4 running
00121: process 2 ready
00122: process 4 ready
00122: process 4 running
00124: process 4 blocked
00124: process 2 running
00126: process 4 ready
00127: process 2 ready
00127: process 2 running
00129: process 2 finished
00129: process 4 running
00132: process 4 ready
00132: process 4 running
00134: process 4 finished
Process 1: Run time 25, Block time 5, Wait time 35
Process 2: Run time 25, Block time 8, Wait time 96
Process 3: Run time 25, Block time 5, Wait time 70
Process 4: Run time 25, Block time 8, Wait time 101
Process 5: Run time 7, Block time 25, Wait time 0
Process 6: Run time 10, Block time 24, Wait time 7
Process 7: Run time 7, Block time 25, Wait time 4
Process 8: Run time 10, Block time 24, Wait time 11
